eiffel-vocabulary/EiffelAnnouncementAcknowledgedEvent.md

```diff
@@ -1,53 +0,0 @@
-<!---
-   Copyright 2017 Ericsson AB.
-   For a full list of individual contributors, please see the commit history.
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
---->
-
-# EiffelAnnouncementAcknowledgedEvent   (AnnA)
-The EiffelAnnouncementAcknowledgedEvent is to declare that an EiffelAnnouncementPublishedEvent has been received & acknowledged by a person or a machine.
-
-## Data Members
-### data.acknowledgement
-__Type:__ String  
-__Required:__ Yes  
-__Description:__ The Acknowledgement response.
-
-### data.user
-__Type:__ String  
-__Required:__ No  
-__Description:__ The identity of the user or entity that acknowledged the EiffelAnnouncementPublishedEvent. 
-
-### data.uri
-__Type:__ String  
-__Required:__ No  
-__Description:__ A URI where further information can be obtained, if applicable. 
-
-## Legal Link Types
-### CAUSE
-__Legal targets:__ Any  
-__Multiple allowed:__ Yes  
-__Description:__ Identifies a cause of the event occurring. SHOULD not be used in conjunction with __CONTEXT__: individual events providing __CAUSE__ within a larger context gives rise to ambiguity. It is instead recommended to let the root event of the context declare __CAUSE__.  
-
-### ACTIVITY_EXECUTION
```

Comment:
If this event should be there, this could be CONTEXT not ACTITVY_EXECUTION

Reply:
File has been deleted.